### PR TITLE
docs(kong-user) fix install notes to switch users

### PR DESCRIPTION
**Pull request by murillopaula**
Now that https://github.com/Kong/kong/pull/6421 is merged, this part of the docs need to be updated.

### CODE REVIEW

File: app/_includes/md/2.2.x/ee-kong-user.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ Amazon Linux 2, CentOS, Ubuntu, and RHEL -->
 
 <div class="alert alert-ee blue">
 <b>Note:</b> When you start Kong, the Nginx master process runs
-as <code>root</code>, and the worker processes run as <code>nobody</code> by
+as <code>root</code>, and the worker processes run as <code>kong</code> by
 default. If this is not the desired behavior, you can switch to the built-in
```
Comment (lena-larionova) [inline app/_includes/md/2.2.x/ee-kong-user.md:7]:
```suggestion
default. If this is not the desired behavior, you can switch the Nginx master process to run on the built-in
```

File: app/_includes/md/ce-kong-user.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
  CentOS, Debian, RedHat, and Ubuntu -->
 
     **Note:** When you start Kong, the Nginx master process runs
-    as `root` and the worker processes as `nobody` by default.
+    as `root` and the worker processes as `kong` by default.
     If this is not the desired behavior, you can switch to the built-in
```
Comment (lena-larionova) [inline app/_includes/md/ce-kong-user.md:6]:
```suggestion
    If this is not the desired behavior, you can switch the Nginx master process to run on the built-in
```

**Review by lena-larionova — COMMENTED** [review]
Minor edit to clarify the difference between the default use of `kong` user and what you're switching.

**Comment by murillopaula** [comment]
@lena-larionova good catch! also, I need to start clicking the 'Add suggestion to bach' so the builds don't trigger for every commit suggestion and we don't waste CPU time 😆 thx, please do a squash and merge as always since we don't want these 'Update <file_path>" commits in the commit history.

**Comment by lena-larionova** [comment]
> please do a squash and merge as always since we don't want these 'Update <file_path>" commits in the commit history.

Always!

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**